finance/bitcoin/Portfile

```diff
@@ -26,22 +26,20 @@ long_description        Bitcoin is a peer-to-peer digital currency. By peer-to-p
 homepage                https://bitcoincore.org/
 master_sites            ${homepage}bin/bitcoin-core-${version}/
 
-checksums               rmd160  b10da11974ccd2b80d49ae4c2f515e9d3a83695d \
-                        sha256  bec2a598d8dfa8c2365b77f13012a733ec84b8c30386343b7ac1996e901198c9 \
-                        size    12498596
+checksums               rmd160  b3b0060c79474128a27c4b6f4049e398497a63b5 \
+                        sha256  ab1d99276e28db62d1d9f3901e85ac358d7f1ebcb942d348a9c4e46f0fcdc0a1 \
+                        size    12754015
 
 depends_build           port:autoconf \
                         port:automake \
                         port:libtool \
                         port:pkgconfig \
-                        port:python311
+                        port:python312
```

Comment:
You can't just change the dependency. You also have to tell the build to use this version of python. Looking through the history of the port, back to when the python dependency was originally added in #1268, I don't see how this has ever worked.

Reply:
By which I mean, yes, the port builds, but I see no indication in the log that python is searched for or found. So either this software doesn't use python, in which case remove the dependency, or it searches for python by various names and uses whichever one it found, in which case fix it to use specifically this python that you've declared a dependency on.

Reply:
> but I see no indication in the log that python is searched for or found

The indication in the log is right here:
> checking for python3.12... /opt/local/bin/python3.12